packages/collaborative-editing-backend/.env.example

```
@@ -0,0 +1,8 @@
+HOST=0.0.0.0
+PORT=1234
+
+MONGODB_URL=mongodb://localhost:27017
+MONGODB_DB=yjs_documents
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGODB_DB key should go before the MONGODB_URL key

(UnorderedKey)